README.md

```diff
@@ -9,6 +9,29 @@ FlashStore is a Key-Value database tool and makes using leveldb more easy for No
 
 ![flash store](https://zixia.github.io/flash-store/images/flash-store.png)
 
+EXAMPLE
+--------
+try the following command
+```shell
+npm install flash-store
+npm run demo
```

Comment:
No, you can not do this.

You might want to try those two commands in a newly created directly and see what will happen.

Reply:
The latest one I change to `npm install`. How about this one?